examples/webpack.config.local.js

```diff
@@ -15,11 +15,12 @@
 
 // avoid destructuring for older Node version support
 const resolve = require('path').resolve;
+const {getOcularConfig} = require('ocular-dev-tools');
```

Comment:
@ibgreen I found that I couldn't run examples without changing this. I noticed the same used pattern in https://github.com/visgl/deck.gl/pull/6183, so I take it that you simply forgot to include that in that PR CC @Pessimistress